Dockerfile

```diff
@@ -1,27 +1,28 @@
-FROM ubuntu:15.10
+FROM ubuntu:14.04
```

Comment:
It's the common practice I've seen for ubuntu based image and `ubuntu:latest` does point to `ubuntu:14.04` (see https://hub.docker.com/_/ubuntu/). But it's arguable.